src/styles/mixins/molecules.scss

```diff
@@ -114,9 +114,12 @@
 /**
  * Base styles for a "panel"-like element, with appropriate
  * padding, heading and close-button styles.
+ *
+ * @param {length} [$rhythm] - An optional value to use for vertical rhythm
+ *   (spacing, vertically)
```

Comment:
I see you're using JSDoc-style syntax here. The meaning is clear, although I don't know if this is an "officially supported" syntax.

Reply:
It's totally a made-up fantasy world use of JSDoc-style comments. However, I wanted to capture this "type" of information and didn't see a built-in way to do it in SCSS source. Open to suggestions.